kotlinx-coroutines-core/common/src/CoroutineScope.kt

```diff
@@ -6,39 +6,34 @@ package kotlinx.coroutines
 
 import kotlinx.coroutines.internal.*
 import kotlinx.coroutines.intrinsics.*
-import kotlin.coroutines.intrinsics.*
 import kotlin.coroutines.*
+import kotlin.coroutines.intrinsics.*
 
 /**
  * Defines a scope for new coroutines. Every coroutine builder
  * is an extension on [CoroutineScope] and inherits its [coroutineContext][CoroutineScope.coroutineContext]
  * to automatically propagate both context elements and cancellation.
  *
+ * The best way to obtain a standalone instance of the scope is [CoroutineScope()] and [MainScope()] factory functions.
```

Comment:
Grammatical number is not balanced. Rephrase to either "The best way is ... _using_ this and that" or "The best _ways_ _are_ ... this and that"